joystick_pub_node:
    ros__parameters:
      scale_linear: 1.0 
      scale_linear_turbo: 2.0

      scale_angular: 1.0
      scale_angular_turbo: 2.0

twist_mux:
  ros__parameters:
    topics:
      navigation:
        topic   : cmd_vel
        timeout : 0.5
        priority: 10
      joystick:
        topic   : cmd_vel_joy
        timeout : 0.5
        priority: 100